src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/SecurityTokenRequest.cs

```diff
@@ -41,6 +47,11 @@ class RequestSecurityTokenResponse
         /// Gets or sets the access token.
         /// </summary>
         public string Token { get; set; }
+
+        /// <summmary>
+        /// Gets or sets the token expiration
+        /// </summary>
+        public int ExpiresOn { get; set; }
```

Comment:
Given that this is just an int I'd suggest being more explicit in the doc comment explaining what this value represents